pkg/subctl/cmd/gather/gather.go

```diff
@@ -65,7 +65,7 @@ var gatherTypeFlags = map[string]bool{
 	"resources": false,
 }
 
-var gatherFuncs = map[string]func(string, Info) bool{
+var gatherFuncs = map[string]func(string, Info) {
```

Comment:
Why was the return value removed? The reason for it is to avoid outputting any status if the module/data type doesn't apply.

Reply:
The idea was to avoid outputting anything if the module/data type doesn't apply (like for `gatherBroker`, _Logs_ doesn't apply).  Either way though I think changing this function signature is really outside the scope of this PR.

Reply:
Changing the signature, I think is a side effect of balancing the status start and end.

Reply:
Then how do we handle the case where the module/data type doesn't apply? Either way this doesn't pertain to the stated purpose of this PR.

Reply:
well, I have reverted the change for now. Lets discuss this offline.